packages/react/src/components/ProgressIndicator/ProgressIndicator.props.ts

```diff
@@ -0,0 +1,27 @@
+import { Size } from '@naturacosmeticos/natds-themes'
+
+export type ProgressIndicatorSize = keyof Pick<Size, 'standard' | 'semi' | 'medium' | 'large'>
+export interface ProgressIndicatorProps {
+  /**
+   * Describe the purpose of Progress Indicator
+   * Read more [here](https://dequeuniversity.com/rules/axe/4.2/aria-progressbar-name?application=axeAPI)
+   */
+  ariaLabel: string
+
+  /**
+   * Show a layer behind the progress indicator
```

Comment:
O que você acha desse nome "Layer"? To vendo no storybook e na UIDOC e nem dá pra ver a diferença, mas entendo que é o background de cor surface atrás do componente, como aparece no pattern do overlay, não é? Acho que vale a gente conversar sobre um nome mais ilustrativo e/ou uma forma de documentar mais fácil de visualizar. Você concorda?

Reply:
+1 pra estabelecer um nome pra essa coisa ai.
em outras libs vejo backdrop
fico pensando se overlay é uma layer por cima e backdrop é uma layer por baixo?
🤔

Reply:
real! sempre vi overlay usado como algo por cima do conteúdo mesmo, tipo uma máscara com opacidade que adiciona uma sombra em cima de imagem, ou aquela cobertura semi transparente que aparece em cima de toda a página quando abre um modal. já backdrop é novo pra mim, mas faz sentido ser essa lógica por baixo hahaha.

nesse caso eu super iria por um background, backgroundLayer, surface, alguma palavra mais usual possível.

mas Yas levantou um ponto: esse componente já tá codado em todas as outras libs e seria 3 breaking changes mudar o nome! 💀